README.md

```diff
@@ -41,16 +41,16 @@ set via environment variables and makes it easy to check external services (like
 * provides generic build and life cycle infrastructure for containerized PHP applications (well prepared for kubernetes)
 * shipped as PHP 7.1.x and 7.2.x flavor
 * there is a (build>base already run / build>base only prepared) flavor for each PHP version
-* general run concept: `docker run --rm -t claranet/php:1.1.15-php7.2.5 <main-section> [subsection] [subsection-args, ...]`
-* overview: `docker run --rm -t claranet/php:1.1.15-php7.2.5 help`
+* general run concept: `docker run --rm -t claranet/php:1.1.16-php7.2.15 <main-section> [subsection] [subsection-args, ...]`
+* overview: `docker run --rm -t claranet/php:1.1.16-php7.2.5 help`
```

Comment:
this one seems to be missing `7.2.5` vs. `7.2.15`

Reply:
Sorry, I've fixed it now